website/docs/docs/dbt-cloud/access-control/self-service-permissions.md

```diff
@@ -18,3 +18,36 @@ dbt Cloud supports two different permission sets to manage permissions for self-
 | Manage billing | ❌ | ✅ |
 | Manage team permissions | ❌ | ✅ |
 | Invite Owners to the account | ❌ | ✅ |
+
+## Read Only vs. Developer License Types
+
+Users configured with Read Only license types will experience a restricted set of permissions in dbt Cloud. If a user is associated with an member permission set and a Read Only seat license, then they will only have access to what a Read-Only seat allows. See [Seats and Users](cloud-seats-and-users) for more information on the impact of licenses on these permissions.
```

Comment:
```suggestion
Users configured with Read Only license types will experience a restricted set of permissions in dbt Cloud. If a user is associated with a _Member_ permission set and a Read Only seat license, then they will only have access to what a Read-Only seat allows. See [Seats and Users](cloud-seats-and-users) for more information on the impact of licenses on these permissions.
```

Reply:
Thanks @drewbanin and nice catch on the indefinite article!

Reply:
@drewbanin @amychen1776 On whether this page is the right home for this, I could see this moving to either Access Control or Enterprise Permissions. But, wondering if the setup steps would fit better in a new "Enterprise SSO Account Onboarding" page? Don't have a super-strong opinion here.

Reply:
ok - i similarly do not feel strongly about the home for this content. I think an Enterprise SSO Account Onboarding page might be a _great_ idea, but we can save that one for later :)

Call it how you see it, and if you do make a change, please feel free to tag me for re-review! If not, this LGTM as-is :)